assets/src/blocks/amp-story/edit.js

```diff
@@ -74,33 +78,175 @@ export default class EditPage extends Component {
+		if ( media.media_type ) {
+			if ( media.media_type === VIDEO_BACKGROUND_TYPE ) {
+				mediaType = VIDEO_BACKGROUND_TYPE;
+			} else {
+				mediaType = IMAGE_BACKGROUND_TYPE;
+			}
+		} else {
+			// For media selections originated from existing files in the media library.
+			if (
+				media.type !== IMAGE_BACKGROUND_TYPE &&
+				media.type !== VIDEO_BACKGROUND_TYPE
+			) {
+				return;
+			}
+
+			mediaType = media.type;
+		}
+
+		this.props.setAttributes( {
+			mediaUrl: media.url,
+			mediaId: media.id,
+			mediaType: mediaType,
+		} );
+
+		if ( IMAGE_BACKGROUND_TYPE === mediaType ) {
+			this.props.setAttributes( { poster: undefined } );
+		}
 	}
 
 	render() {
-		const props = this.props;
-		const { attributes } = props;
+		const { attributes, media, setAttributes } = this.props;
+
+		const { backgroundColor, mediaId, mediaType, mediaUrl, focalPoint, poster } = attributes;
+
+		const instructions = <p>{ __( 'To edit the background image or video, you need permission to upload media.', 'amp' ) }</p>;
+
+		const style = {
+			backgroundColor: backgroundColor,
+			backgroundImage: IMAGE_BACKGROUND_TYPE === mediaType && mediaUrl ? `url(${ mediaUrl })` : undefined,
+			backgroundPosition: IMAGE_BACKGROUND_TYPE === mediaType && focalPoint ? `${ focalPoint.x * 100 }% ${ focalPoint.y * 100 }%` : 'cover',
+			backgroundRepeat: 'no-repeat',
+		};
+
+		if ( VIDEO_BACKGROUND_TYPE === mediaType && poster ) {
+			style.backgroundImage = `url(${ poster })`;
+		}
 
 		return (
 			<Fragment>
 				<InspectorControls key="controls">
 					<PanelColorSettings
-						title={ __( 'Background Color Settings', 'amp' ) }
+						title={ __( 'Color Settings', 'amp' ) }
 						initialOpen={ false }
 						colorSettings={ [
 							{
-								value: attributes.backgroundColor,
-								onChange: this.onChangeBackgroundColor,
+								value: backgroundColor,
+								onChange: ( value ) => setAttributes( { backgroundColor: value } ),
 								label: __( 'Background Color', 'amp' ),
 							},
 						] }
 					/>
+					<PanelBody title={ __( 'Background Media', 'amp' ) }>
+						<Fragment>
+							<BaseControl>
+								<MediaUploadCheck fallback={ instructions }>
+									<MediaUpload
+										onSelect={ this.onSelectMedia }
+										allowedTypes={ ALLOWED_MEDIA_TYPES }
+										value={ mediaId }
+										render={ ( { open } ) => (
+											<Button isDefault isLarge onClick={ open } className={ 'editor-amp-story-page-background' }>
+												{ mediaUrl ? __( 'Edit Media', 'amp' ) : __( 'Upload Media', 'amp' ) }
+											</Button>
+										) }
+									/>
+								</MediaUploadCheck>
+								{ !! mediaId &&
+								<MediaUploadCheck>
+									<Button onClick={ () => setAttributes( { mediaUrl: undefined, mediaId: undefined, mediaType: undefined } ) } isLink isDestructive>
+										{ VIDEO_BACKGROUND_TYPE === mediaType ? __( 'Remove video', 'amp' ) : __( 'Remove image', 'amp' ) }
+									</Button>
```

Comment:
Maybe it would be good to add a suggested image / video ratio based on the [best practices](https://www.ampproject.org/docs/fundamentals/amp_story_best_practices#specify-a-poster-attribute) so that the user would have an idea which ratio to aim for the best experience. Thoughts?

Reply:
Added a help text:

<img width="286" alt="screenshot 2019-02-28 at 17 10 17" src="https://user-images.githubusercontent.com/841956/53580388-c8f49880-3b7b-11e9-8cce-677ab2d5134f.png">